examples/agentInteractionsDemo/index.html

```diff
@@ -0,0 +1,197 @@
+                            Start: {{moment(conv.startTime).format('MMM Do YYYY, h:mm:ss A')}}
+                        </p>
+                        <p v-if="conv.endTime" class="card-text mb-2 text-muted">
+                            End: &nbsp {{moment(conv.endTime).format('MMM Do YYYY, h:mm:ss A')}}
+                        </p>
+                        <div class="dropdown dropright">
+                            <button v-if="convEvalMap.has(conv.id)"
+                                    id="dropdownMenuBtn"
+                                    class="view-evaluations btn btn-secondary dropdown-toggle"
+                                    data-toggle="dropdown"
+                                    aria-haspopup="true"
+                                    aria-expanded="false">
+                                View Evaluations
+                            </button>
+                            <div class="dropdown-menu" aria-labelledby="dropdownMenuBtn">
+                                <a v-if="convEvalMap.has(conv.id)"
+                                    v-for="eval in convEvalMap.get(conv.id)"
+                                    @click="viewEvaluation(conv.id, eval.id)"
+                                    class="dropdown-item">
+                                    {{eval.evaluationForm.name}}
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </script>
+
+    <!-- template for the conversations component -->
+    <script type="text/x-template" id="tester-template">
+        <section class="manual-tester">
+            <form @submit.prevent="getConversationOrEvaluation">
+                <h2>Manual Tester</h2>
+                <div class="form-group conversation-id">
+                    <label for="conversation-id">Conversation ID:</label>
+                    <input type="text"
+                        class="form-control"
+                        id="conversation-id"
+                        v-model="convId">
+                </div>
+                <div class="form-group evaluation-id">
+                    <label for="evaluation-id">Evaluation ID:</label>
+                    <input type="text"
+                        class="form-control"
+                        id="evaluation-id"
+                        v-model="evalId">
+                </div>
+                <div class="form-group">
+                    <button type="submit"
+                        class="btn btn-primary"
+                        v-bind:class="{'disabled':!this.convId}">
+                        Submit
+                    </button>
+                </div>
+            </form>
+        </section>
+    </script>
+
+    <!-- app -->
+    <div id="app">
+        <!-- authenticating placeholder-->
+        <authenticating :error-message="errorMessage"
```

Comment:
Should this be hidden once a user is authenticated?  It will be empty so this isn't critical.  Just thinking "out loud"

Reply:
ah, just noticed that this is not hidden because non-auth messages could be shown here.  We could elevate this out of the authenticating component to make this more clear.

Reply:
I am thinking about how to do this. I put this inside authenticating component because I thought it would be more organized(as all parts are components).